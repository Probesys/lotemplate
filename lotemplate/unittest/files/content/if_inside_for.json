{
  "tutu": {
    "type": "array",
    "value": [
      {
        "type": "person",
        "firstName": "perso 1",
        "lastName": "string 1",
        "address": {
          "street1": "8 rue de la paix",
          "street2": "",
          "zip": "75008",
          "city": "Paris",
          "state": "Ile de France"
        }
      },
      {
        "type": "person",
        "firstName": "perso 2",
        "lastName": "lastname with < and >",
        "address": {
          "street1": "12 avenue Jean Jaurès",
          "street2": "",
          "zip": "38000",
          "city": "Grenoble",
          "state": "Isère"
        }
      },
      {
        "type": "company",
        "name": "my_company",
        "address": {
          "street1": "12 avenue Gambetta",
          "street2": "",
          "zip": "38000",
          "city": "Grenoble",
          "state": "Isère"
        }
      }
    ]
  },
  "outsidefor": {
    "type": "text",
    "value": "foo"
  }
}