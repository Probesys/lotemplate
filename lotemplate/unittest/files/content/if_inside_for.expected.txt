
Tests of if inside for statements


Associate number 0
first name : perso 1 string 1


Address : 
8 rue de la paix
75008 Paris

inside for :
before_should display_after
before__after


Associate number 1
first name : perso 2 lastname with < and >


Address : 
12 avenue Jean Jaurès
38000 Grenoble

inside for :
before_should display_after
before__after


Associate number 2

company name : my_company

Address : 
12 avenue Gambetta
38000 Grenoble

inside for :
before_should display_after
before__after



outside for

before_should display_after
before__after

Test if on the forindex

Associate number 1
first name : perso 2 lastname with < and >

Address : 
12 avenue Jean Jaurès
38000 Grenoble

End of file
